pegasus/sites.v3/code.org/public/educate/csa.haml

```diff
@@ -101,7 +101,7 @@ theme: responsive_full_width
 
 %section#csa-newsletter-signup{style: "padding-top: 2rem"}
   .wrapper
-    = view :curriculum_subscribe_form, subscribe_form_section_desc: hoc_s(:subscribe_form_section_desc_csa), form_src: "http://go.pardot.com/l/153401/2021-02-19/p291ns"
+    = view :subscribe_form, subscribe_form_section_title: hoc_s(:subscribe_form_section_title), subscribe_form_section_desc: hoc_s(:subscribe_form_section_desc_csa), form_height: "375px", form_src: "http://go.pardot.com/l/153401/2021-02-19/p291ns"
```

Comment:
Related to [this comment](https://github.com/code-dot-org/code-dot-org/pull/55039/files#r1402533719).